Source/WebCore/accessibility/AXCoreObject.cpp

```diff
@@ -298,7 +298,12 @@ AXCoreObject::AXValue AXCoreObject::value()
 
     if (isColorWell()) {
         auto color = convertColor<SRGBA<float>>(colorValue()).resolved();
-        return makeString("rgb "_s, String::numberToStringFixedPrecision(color.red, 6, TrailingZerosPolicy::Keep), ' ', String::numberToStringFixedPrecision(color.green, 6, TrailingZerosPolicy::Keep), ' ', String::numberToStringFixedPrecision(color.blue, 6, TrailingZerosPolicy::Keep), ' ', color.alpha == 1.0f ? "1"_s : String::numberToStringFixedPrecision(color.alpha, 6, TrailingZerosPolicy::Keep));
+        auto serializedColorChannel = [](float number) {
```

Comment:
Given the really narrow scope here, where the lambda is local to this one block, I’d be tempted to go further and give this a less descriptive one word name, like just `channel`.